modules/domain/README.md

```diff
@@ -1,27 +1,17 @@
-# Module CIS Domain
+# CIS Domain Module
 
-This module is used to manage CIS Instance, CIS Domains, CIS DNS Records.
+This module is used to manage CIS Domain.
```

Comment:
Perhaps a wording along the lines of "configuring the domain that is assigned to this CIS instance" to distinguish from the DNS subdomain registration (in dns module)

Reply:
Made some changes, please check.